csm/conf/configure.py

```diff
@@ -371,16 +369,13 @@ def _create_csm_ldap_user(self):
         ldap_user = Conf.get(const.CSM_GLOBAL_INDEX, const.LDAP_AUTH_CSM_USER)
         csm_ldap_secret =  Conf.get(const.CSM_GLOBAL_INDEX, const.LDAP_AUTH_CSM_SECRET)
         csm_admin_ldap_password = self._fetch_ldap_password(csm_ldap_secret)
-        _output, _err, _rc = self._run_ldap_cmd(f'slappasswd -s {csm_admin_ldap_password}')
-        csm_admin_ldap_password = _output.split('\n')[0]
         ldap_url = Setup._get_ldap_url()
         tmpl_init_data = Text(const.CSM_LDAP_ADMIN_USER_LDIF).load()
         tmpl_init_data = tmpl_init_data.replace('<base-dn>',base_dn)
         tmpl_init_data = tmpl_init_data.replace('<csm-admin-user>',ldap_user)
         tmpl_init_data = tmpl_init_data.replace('<csm-admin-password>',csm_admin_ldap_password)
         Text(const.CSM_LDAP_ADMIN_FILE_PATH).dump(tmpl_init_data)
-        self._run_ldap_cmd(f'ldapadd -x -D {bind_base_dn} -w {_rootdnpassword} -f {const.CSM_LDAP_ADMIN_FILE_PATH}\
-        -H {ldap_url}')
+        self._perform_ldif_parsing(const.CSM_LDAP_ADMIN_FILE_PATH, bind_base_dn, _rootdnpassword)
 
     def _run_ldap_cmd(self, cmd):
```

Comment:
Please check if this function required now ?

Reply:
Modifications done